spyder/utils/site/sitecustomize.py

```diff
@@ -555,7 +555,10 @@ def __init__(self, namelist=None, pathlist=None):
             namelist = []
         spy_modules = ['sitecustomize', 'spyder', 'spyderplugins']
         mpl_modules = ['matplotlib', 'tkinter', 'Tkinter']
-        self.namelist = namelist + spy_modules + mpl_modules
+        other_modules = ['pytorch']
+        if PY2:
+            other_modules.append('astropy')

```

Comment:
Please add comment with a reference to the issues that lead us to blacklist each of these modules, one comment per module.

Thanks!